### PR TITLE
Python docs

### PR DESCRIPTION
**Pull request by hadarhg**
- Add Quickstart for Python
- Add Python to WordCount doc

### CODE REVIEW

**Comment by hadarhg** [comment]
R: @aaltay

File: src/get-started/wordcount-example.md [hunk 1]
```diff
@@ -60,12 +68,20 @@ You can specify a runner for executing your pipeline, such as the `DataflowRunne
     //   options.setRunner(FlinkRunner.class);
 ```
 
+```py
+{% github_sample /apache/beam/blob/python-sdk/sdks/python/apache_beam/examples/snippets/snippets.py tag:examples_wordcount_minimal_options
+%}```
```
Comment (hadarhg) [inline src/get-started/wordcount-example.md]: Snippet needs to be updated, see https://issues.apache.org/jira/browse/BEAM-1322

File: src/get-started/quickstart-py.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+### Create and activate a virtual environment
+
+A virtual environment is a directory tree containing its own Python distribution. To create a virtual environment, create a directory and run:
+
+```
+virtualenv /path/to/directory
+```
+
+A virtual environment needs to be activated for each shell that is to use it.
```
Comment (aaltay) [inline src/get-started/quickstart-py.md]: Could you link to `https://virtualenv.pypa.io/en/stable/userguide/#activate-script` for the activated. So users of non-Bash shells know what do (i.e. Windows).

We might even explicitly say for other shells see virtualenv documentation and link.

File: src/get-started/wordcount-example.md [hunk 3]
```diff
@@ -79,52 +95,73 @@ The Minimal WordCount pipeline contains five transforms:
...
+                if (!word.isEmpty()) {
+                    c.output(word);
+                }
+            }
+        }
+    }))
+    ```
+
+    ```py
+    # The Flatmap transform is a simplified version of ParDo.
```
Comment (aaltay) [inline src/get-started/wordcount-example.md]: Why this comment about the FlatMap is not part of the snippet?

Comment (hadarhg) [inline src/get-started/wordcount-example.md]: I would have preferred it to be text but we don't have the ability to put text inside the language toggles yet so I put it in as a comment temporarily.

**Comment by hadarhg** [comment]
Thanks, Ahmet. Updated, PTAL. Details for Travis failure [here](https://travis-ci.org/apache/beam-site/builds/195407964). Not sure what to do about that, but it seems like it can be ignored. Wdyt?

**Comment by aaltay** [comment]
LGTM. Thank you!

File: src/get-started/quickstart-py.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+[Install pip](https://pip.pypa.io/en/stable/installing/), Python's package manager. Check if you already have `pip` installed by running `pip --version`. After installation, verify that you have `pip` version 7.0.0 or newer. To update `pip`, run the following command:
+
+```
+pip install -U pip
+```
+
+### Install virtualenv
+
+It is recommended that you install a [Python virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/)
```
Comment (davorbonaci) [inline src/get-started/quickstart-py.md]: I'd simplify -- we don't need to give so many commands. Applies to all steps.

If you don't have XYZ version ABC or newer, install it:
   command to install

**Comment by davorbonaci** [comment]
Merging based on @aaltay's LGTM and a quick skim over it.

**Pull request closed without merging**